spec/unit/shoulda/matchers/active_record/define_enum_for_matcher_spec.rb

```diff
@@ -57,19 +81,26 @@ def self.statuses; end
         end
 
         it "accepts a record where the attribute is not defined as an enum" do
-          message = %{Expected #{record_with_array_values.class} to define :#{non_enum_attribute} as an enum with ["open", "close"]}
+          message = %{Expected #{record_with_array_values.class} to define :#{non_enum_attribute} as an enum with ["open", "close"] and store the value in a column with an integer type}
+
+          assertion = lambda do
+            expect(record_with_array_values).
+              to define_enum_for(non_enum_attribute).with(["open", "close"])
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.